src/sentry/models/apiscopes.py

```diff
@@ -73,10 +73,16 @@ class Meta:
     # Human readable list of scopes
     scope_list = ArrayField(of=models.TextField)
 
-    def get_scopes(self):
+    def get_scopes(self) -> List[str]:
+        """
+        Returns a list of the token's scopes in alphabetical order.
+        """
         if self.scope_list:
-            return self.scope_list
-        return [k for k, v in self.scopes.items() if v]
+            return sorted(self.scope_list)
+        return sorted([k for k, v in self.scopes.items() if v])
```

Comment:
We want to return a sorted list so our responses have scopes in alphabetical order

Reply:
this is only for aesthetic purposes, right?

Reply:
Yes, this makes it a lot easier to read scopes in response bodies